ansible/roles-infra/infra-ec2-template-generate/defaults/main.yml

```diff
@@ -97,270 +97,270 @@ aws_default_image: RHEL75
 
 aws_ami_region_mapping:
   eu-north-1:
-    RHEL82GOLD: ami-0c14a12dadebed14f # RHEL-8.2_HVM-20200803-x86_64-0-Access2-GP2
```

Comment:
These comments requiring two spaces... pretty sure we want to change our `.yamllint` config to allow single space here.

Reply:
fixed